cloud/deploys/dev_azure/README.md

```diff
@@ -4,8 +4,42 @@ app. We intentionally don't use a shared state with this terraform directory
 to allow for everyone to have their own app. 
 
 The process to deploy is similar
-to the staging process but we do not have to do the shared state set up. It 
-should be `terraform init` and then `terraform apply`
+to the staging process but run from the cloud/deploys/dev_azure folder. 
+
+# Running the dev deploy
+## Copy the civiform_config.example.sh 
+Copy the civiform_config.example.sh into civiform_config.sh and
+ change the required variables. 
+
+## Setup Login Radius For Local Development
+Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click
+configure a civiform integration. Choose the outbound SSO Saml.
+
+From there add an app with Sp initiated login and pick a name (this gets put
+into the config as `LOGIN_RADIUS_SAML_APP_NAME`).
+
+To generate the private key for the form run, cat the file and put into dashboard.
+```
+openssl genrsa -out private.key 2048
+```
+
+For generating the cert run, cat the file and put into dashboard.
+```
+openssl req -new -x509 -key private.key -out certificate.cert -days 365 -subj /CN=civiform-staging.hub.loginradius.com
+```
+
+We need to copy the details from a previous working setup in login radius 
+once we set up the certs so look back at the staging one to fill out.
+
+## Source and run 
+After that you can start the setup by running:
+
+```
+source cloud/deploys/dev_azure/civiform_config.sh
+cloud/shared/bin/setup  
```

Comment:
suggestion (blocking): this should be a script in `cloud/deploys/dev_azure/bin/setup`, no?

Reply:
i meannnn it can be but was being a lil lazy here

Reply:
good for consistency with our other scripts :-)